src/markdown.ts

```diff
@@ -36,10 +35,8 @@ export interface MarkdownPage {
   code: MarkdownCode[];
 }
 
-interface ParseContext {
+export interface ParseContext {
   code: MarkdownCode[];
-  startLine: number;
-  currentLine: number;
```

Comment:
This code has been dead for a while, so I’m removing it for now. I assume at some point we’ll get back to implementing better line numbers for errors, but better to delete the code if it’s not currently used.